Setting up starter files, folders and database connection

bizip opened the pull request:
#### What does this PR do?

- Created a startup folder structure as required

- Added configuration file like .env and .babelrc

- Created a connection to MongoDB

#### Description of Task to be completed

All task described above is completed 

#### How should this be manually tested?
Step 1: Go to https://github.com/bizip/Pascal-capstone-project

Step 2: Click on code (Button)
![image](https://user-images.githubusercontent.com/52130946/93663232-8698f200-fa66-11ea-9222-e7517065af61.png)


Step 3: Copy string of code

Step 4: Open your terminal on your local machine and change directory to desktop

Step 5: type git clone and paste the link then press enter

Step 6: Type cd pascal-capstone-project

Step 7: Run npm install and all package will be installed

Step 8: type npm start

#### What are the relevant Trello story?

https://trello.com/c/7ahHIUal

bizip committed:
ft(All query/question): display  list of all questions

-Displayed all list of query/question on one page
-Created delete botton to delete single question
-Added a well displayed message when delete action is done

[ Done ]

bizip committed:
bg(fixed delete): fixed issue on delete button

[ done ]

bizip committed:
ft(Send Question): Send question in firebase

-Connected contact form to firebase
-Well displayed message when a message is successfull sent
-Reset form once action is done using promises

[ Done ]

bizip committed:
ft(Post article): Post new aricle in firebase

-Connected new article form to firebase
-Displayed a success message once aricle is successfully sent to firebase

[ Done ]

bizip committed:
ch(Startup files ): Setting up database connection

-Created startup folder structure as required
-Added configuration file like .env, .babelrc
-Created a connection to MongoDB

[ Done ]

mukunzidd commented on .env:
```diff
@@ -0,0 +1,3 @@
+HOST=localhost
+PORT=8000
+DB_URL='mongodb+srv://bizimungu:0728038pp.PP@agakiriro.0xgn1.mongodb.net/blog?retryWrites=true'
```
Your .env file should be ignore by git. i.e put it in .gitignore, otherwise everyone on the internet will be able see your secrets (passwords, ports, and keys) which is something that you don't wanna do.

mukunzidd commented on UI/html/question.html:
```diff
@@ -75,32 +75,33 @@ <h1>Subscribe to for news letter</h1>
                 <!-- <div class="article"> -->
-                <h4><b>Benimana Eric</b></h4>
+                <!-- <h4><b>Benimana Eric</b></h4>
                 <p class="article__text">Prosciutto beef ribs strip steak, biltong chislic tri-tip shankle. Biltong drumstick alcatra, shankle leberkas brisket pork belly pork chop. Rump andouille pork chop, beef ribs fatback t-bone brisket shankle tur salami short ribs corned
                     beef tongue. <br>
                     <p id="like__icon"><b>bizip04@gmail.com</b>&nbsp;&nbsp;&nbsp;&nbsp;</p>
 
                     <p id="like__icon"><b>Delete</b></p>
 
```
Remove unused code

mukunzidd commented on UI/html/question.html:
```diff
@@ -75,32 +75,33 @@ <h1>Subscribe to for news letter</h1>
+                </p> -->
                 <!-- </div> -->
             </div>
 
-            <div class="item" id="single__article">
-                <!-- <div class="article"> -->
-                <h4><b>Benimana Eric</b></h4>
+            <!-- <div class="item" id="single__article"> -->
+            <!-- <div class="article"> -->
+            <!-- <h4><b>Benimana Eric</b></h4>
                 <p class="article__text">Prosciutto beef ribs strip steak, biltong chislic tri-tip shankle. Biltong drumstick alcatra, shankle leberkas brisket pork belly pork chop. Rump andouille pork chop, beef ribs fatback t-bone brisket shankle tur salami short ribs corned
                     beef tongue. <br>
                     <p id="like__icon"><b>bizip04@gmail.com</b>&nbsp;&nbsp;&nbsp;</p>
                     <p id="like__icon"><b>Delete</b></p>
 
```
same here

bizip committed:
Bg(Working on feedback): Added some file in gitignore

-Added .env file in git .ignore file
-removed UI folder
-Removed some unused code

[ Done ]
